### PR TITLE
Add a log warning if the claim value does not exist

### PR DESCRIPTION
**Pull request by trunges21**
# Explain your changes

Add a log warning if the claim value does not exist

# Checklist

- [ ] I have read the [“Pull requests” section](https://github.com/kinde-oss/.github/blob/489e2ca9c3307c2b2e098a885e22f2239116394a/CONTRIBUTING.md#pull-requests) in the [contributing guidelines](https://github.com/kinde-oss/.github/blob/489e2ca9c3307c2b2e098a885e22f2239116394a/CONTRIBUTING.md).
- [ ] I agree to the terms within the [code of conduct](https://github.com/kinde-oss/.github/blob/489e2ca9c3307c2b2e098a885e22f2239116394a/CODE_OF_CONDUCT.md).

🛟 _If you need help, consider asking for advice over in the [Kinde community](https://thekindecommunity.slack.com)._

### CODE REVIEW

**Comment by mafhaam** [comment]
This works as expected, a warning appears in the console only, if the requested claim doesn’t exist.

File: Sources/KindeSDK/Auth/Auth.swift [hunk 1]
```diff
@@ -51,6 +52,9 @@ public final class Auth {
         guard let params = tokenToParse?.parsedJWT else {
             return nil
         }
+        if !params.keys.contains(key) {
+            os_log("The claimed value of \"%@\" does not existed in your token", log: .default, type: .error, key)
```
Comment (rairaman) [inline Sources/KindeSDK/Auth/Auth.swift:56]: @trunges21 Just a typo.

Can you pls change the log text to:

"The claim value of \"%@\" does not exist in your token"

**Review by rairaman — COMMENTED** [review]
Just the 1 small fix please

File: Sources/KindeSDK/Auth/Auth.swift (hunk 1 above)
Comment (trunges21) [reply Sources/KindeSDK/Auth/Auth.swift]: Yes, I updated

**Review by rairaman — APPROVED** [review]

**Pull request merged**